src/tools/yarprobotstatepublisher/src/main.cpp

```diff
@@ -35,9 +35,11 @@ int main(int argc, char *argv[])
     if (rf.check("help"))
     {
         cout<<"Options"<<endl;
-        cout<<"\t--robot                <robot-name>: robot name"<<endl;
+        cout<<"\t--namePrefix             <name-prefix>: prefix of the yarprobotstatepublisher ports"<<endl;
+        cout<<"\t--tfPrefix               <tf-prefix>: prefix of the published TFs"<<endl;
```

Comment:
The default value of `tfPrefix`  should be documented in  this help.

Reply:
https://github.com/robotology/idyntree/pull/538/commits/2014337cc5eebe96c22e23f33a5948e4d3c62f8d